src/main/assembly/zip-CIEL-dictionary.sql

```diff
@@ -2,7 +2,7 @@
 	xmlns="http://maven.apache.org/plugins/maven-assembly-plugin/assembly/1.1.0"
 	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
 	xsi:schemaLocation="http://maven.apache.org/plugins/maven-assembly-plugin/assembly/1.1.0 http://maven.apache.org/xsd/assembly-1.1.0.xsd">
-	<id>zip-mvp-data</id>
+	<id>zip-CIEL-data</id>

```

Comment:
Ditto

Reply:
This should be `zip-ciel-data` and ideally the file is called `zip-ciel-data.xml` rather than `zip-CIEL-dictionary.sql`.